src/git/blame.ts

```diff
@@ -283,8 +287,10 @@ export class GitBlame {
         const commitUrl = Property.get("commitUrl") || "";
         const origin = await this.getOriginOfActiveFile();
         const projectName = this.projectNameFromOrigin(origin);
+        const remoteUrl = GitBlame.stripGitRemoteUrl(await this.getRemoteUrl());
```

Comment:
If possible we should try to do `this.getOriginOfActiveFile` and `this.getRemoteUrl` in parallel. That would mean changing to something like this:

```suggestion
        const remote = this.getRemoteUrl();
        const origin = await this.getOriginOfActiveFile();
        const projectName = this.projectNameFromOrigin(origin);
        const remoteUrl = GitBlame.stripGitRemoteUrl(await remote);
```

or perhaps cleaning the URL inside `this.getRemoteUrl` and do the await in the replace call further down.

Reply:
done.